Domain/Exceptions/ConfigurationExceptions/PublicPaymentKeyNotFoundException.cs

```diff
@@ -0,0 +1,5 @@
+﻿namespace Ipe.Domain.Errors;
+public class PublicPaymentKeyNotFoundException : BaseException
+{
+    public PublicPaymentKeyNotFoundException() : base("017", "Public payment key not found") { }
```

Comment:
Para remover os Code Smells, precisamos adequar o **BaseException** e este **PublicPaymentKeyNotFoundException**

Seguir exemplos dos métodos protecteds:

https://github.com/the-web-forest/Bonsai/blob/master/Domain/Exceptions/BaseException.cs
https://github.com/the-web-forest/Bonsai/blob/master/Domain/Exceptions/PaymentDeniedException.cs